linux_os/guide/system/crypto_policies/bind_obeys_crypto_policy/oval/shared.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="compliance" id="bind_obeys_crypto_policy" version="1">
+    <metadata>
+      <title>The BIND library obeys system-wide crypto policy.</title>
+      <affected family="unix">
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>The BIND library shouldn't override system-wide crypto policy setting.</description>
```

Comment:
Description should probably be `<description>BIND should be configured to use the system-wide crypto policy setting.</description>`